src/state/pools/index.ts

```diff
@@ -205,6 +251,17 @@ export const PoolsSlice = createSlice({
   name: 'Pools',
   initialState,
   reducers: {
+    setPoolZeroPublicData: (state, action) => {
```

Comment:
Shouldn't it be better to make this generic to set any preferred pool data to state?